### ISSUE TITLE
Need to reduce the font size in plots or make the size configurable

### ISSUE THREAD

**Issue by georgebzheng** [TASBE/TASBEFlowAnalytics#429]
We have recently doubled the length of sample_id to make it unique across all experiments.  An unintended consequence is that now only part of the sample_id is included in the plot title since TASBE uses a fixed font size of 7 to generate the title.  Is it possible to make the title font size smaller or, even better, configurable so we can fit the whole sample_id in the title?

**Comment by jakebeal** [comment]
A simple solution you can use right now is to instead make the plots larger. Would that give you what you want?

The variables to set in TASBEConfig are:
-  `plots.graphPlotSize` (default: `[6 4]`)
- `plots.graphPlotSize` (default: `[5 5]`)

**Comment by georgebzheng** [comment]
Here is the value I used in the TASBEConfig section of color_model_parameters.json and nothing has changed, i.e., the plot size is staying the same.  Am I setting this correctly?

        "TASBEConfig": {
            "sizebeads": {
                "rangeMin": 7,
                "rangeMax": 9.28,
                "peakThreshold": 1500,
                "forceFirstPeak": 1
            },
            "beads": {
                "rangeMin": 6,
                "rangeMax": 10,
                "peakThreshold": 1000
            },
            "heatmapPlottype": "contour",
            "plots": {
                "plotPath": "plots",
                "graphPlotSize": [12, 8]
            }
        },

**Comment by jakebeal** [comment]
Looks like the beads by default override that preference (they prefer a different ratio), so you'll also want to set `beads.plotSize` and `sizebeads.plotSize`.  That will do it for the beads.

Autofluorescence plots apparently aren't using the parameter, so I'll fix that bug... and I found a note in the batch histogram plot that it's ignoring the parameter and needs to be fixed... if we'd been using either of the other analysis models it would be working, but apparently we've hit the sweet spot of places where TASBE wasn't yet updated, waiting for a chance to do it all right with #140.

I'll bump this up in priority.  How urgent a problem is this? (i.e., is this a "nice to have" or "friction in your work" or "blocker")

**Comment by georgebzheng** [comment]
Jake, I just tried using plotSize and as you explained, it only affected the sizebeads.  We are re-processing all datasets starting with non yeast gates, which we will incorporate Dan's live/dead analysis once the code is ready.  Until this is fixed, all sample plots will have title showing partial sample_id.

**Comment by jakebeal** [comment]
So good to have soon, but not an emergency.

**Comment by georgebzheng** [comment]
Yes.

**Comment by jakebeal** [comment]
Configurations are now responding to plot resizing commands.
